x-pack/test/functional_enterprise_search/README.md

```diff
@@ -0,0 +1,33 @@
+# Enterprise Search Functional E2E Tests
+
+## Running these tests
+
+Follow the [Functional Test Runner instructions](https://www.elastic.co/guide/en/kibana/current/development-functional-tests.html#_running_functional_tests).
+
+There are two suites available to run, a suite that requires a Kibana instance without an `enterpriseSearch.host`
+configured, and one that does. The later also [requires a running Enterprise Search instance](#enterprise-search-requirement), and a Private API key
+from that instance set in an Environment variable.
+
+Ex.
+
+```
+# Run specs from the x-pack directory
+cd x-pack
+
+# Run tests that require enterpriseSearch.host variable
+APP_SEARCH_API_KEY=[use private key from local App Search instance here] node scripts/functional_tests --config test/functional_enterprise_search/with_host_configured.config.ts
+
+# Run tests that do not require enterpriseSearch.host variable
+node scripts/functional_tests --config test/functional_enterprise_search/without_host_configured.config.ts
+```
+
+## Enterprise Search Requirement
+
+These tests will not currently start an instance of App Search automatically. As such, they are not run as part of CI and are most useful for local regression testing.
+
+Requirements for Enterprise Search:
+
+- Running on port 3002 against a separate Elasticsearch cluster.
+- Elasticsearch must have a platinum or greater level license (or trial).
```

Comment:
Note to self - if missing a >platinum license, you'll get a rather mysterious error of `SyntaxError: Unexpected end of JSON input`

Reply:
Ugh, yeah. If we decide we want to keep these tests around I think I may just drop a docker-compose file in that will bring up the stack configured correctly.

Reply:
So you don't even have to think about this stuff.